reporting/impl/src/main/resources/reports/templates/techReport-punchCard.ftl

```diff
@@ -0,0 +1,233 @@
+                       maximumsPerTag       // Map<String, Integer>
+                    }
+                <#assign stats = getTechReportPunchCardStats() />
+                -->
+
+                <#-- A precomputed matrix - map of maps of maps, boxTag -> rowTag -> project -> techName -> TechUsageStat.
+                     Map<String, Map<String, Map<Long, Map<String, TechReportService.TechUsageStatSum>>>> -->
+                <#assign sortedStatsMap = sortTechUsageStats() />
+
+
+                <table class="technologiesPunchCard">
+                    <tr class="headersSector">
+                        <td></td>
+                        <#list sectorTags as sector>
+                            <td colspan="${ (iterableToList(sector.designatedTags)?size-1)?c}" class="sector${sector.title}">${sector.title}</td>
+                        <#else>
+                            <td>No technology sectors defined.</td>
+                        </#list>
+                        <td colspan="3" class="sectorStats">Stats</td>
+                    </tr>
+                    <tr class="headersGroup">
+                        <td></td>
+                        <#list sectorTags as sectorTag >
+                            <#list sectorTag.designatedTags as boxTag >
+                                <#if !isTagUnderTag(boxTag, sillyTagsParent) >
+                                    <#assign techsOrder = techsOrder + [boxTag] />
+                                    <td class="sector sector${sectorTag.title}"><div>${boxTag.title!}</div></td>
+                                </#if>
+                            </#list>
+                        </#list>
+                        <td class="sector sectorStats sizeMB"><div>Size (MB)</div></td>
+                        <td class="sector sectorStats libsCount"><div>Libraries</div></td>
+                        <td class="sector sectorStats storyPoints"><div>Mandatory (SP)</div></td>
+                    </tr>
+
+
+                    <#list inputApplications as appProject> <#-- ProjectModel -->
+                    <tr class="app">
+                        <td class="name">
+                            <#assign boxReport = reportModel.appProjectIdToReportMap[appProject.asVertex().id?c] > <#-- TechReportModel -->
+                            <a href="${boxReport.reportFilename}">
+                                <#-- For virtual apps, use name rather than the file name. -->
+                                ${ (appProject.projectType! = "VIRTUAL" && appProject.name??)?then(
+                                        appProject.name,
+                                        appProject.rootFileModel.fileName)}
+                            </a>
+                        </td>
+                        <#list sectorTags as sectorTag>
+                            <#list sectorTag.designatedTags as boxTag>
+                                <#if !isTagUnderTag(boxTag, sillyTagsParent) >
```

Comment:
Add comment: This prevents the silly tags from the techReport-hierarchy.tags.xml to appear as a column or box.